src/pages/gd/Stake/index.tsx

```diff
@@ -483,8 +483,8 @@ export default function Stakes(): JSX.Element | null {
                     )}
                 </Modal>
             </StakesSC>
-            {network !== 'production' ? 
-              <Savings network={DAO_NETWORK.FUSE} chainId={chainId} /> : <></>
+            {network !== 'production' && library ? 
+              <Savings network={network} chainId={chainId} /> : <></>
```

Comment:
i've asked that you dont pass network or chainid because you can get them via hooks in Savings

Reply:
added hook for network

Reply:
you still keep passing network to other sub components why?

Reply:
has been removed now, moved to be handled in sdk